### DISCUSSION TITLE
BIOS R1.14.0 where?

### DISCUSSION THREAD

**Discussion by EmbCGuru** [R3NE07/Futro-S740#55] [General]
The Fujitu FJ-ISS-2024-011810 lists a new S740 BIOS, R1.14.0, which should mitigate some security problems, FJ-ISS-2024-011810.

The BISO is not available as a download, or I didn't find it.

**Comment by MichlW** [comment]
BIOS R1.14.0 for D3544-A1:
https://support.ts.fujitsu.com/IndexDownload.asp?SoftwareGuid=FF511B8D-49B1-497C-88B9-E358E2A907B5

20/Dec/2024
·       New GLK Microcodes implemented:  M01706A1_00000040
·       New Intel TXE FW: 4.0.55.2201
·       FIX: TXE FW is not updated with normal update by DeskFlash or Binary Flash by EfiBinFlash.
·       FIX: EFI Shell USB stick is not detected as boot device if FUJITSU Update Utility is present on USB
·       FIX: GLK - FWHunt - potential security issue - BIOS R1.13
·       Update: Security Advisory for GeminiLake
·       Component Updates
·       IPU 2022.3 implemented.
·       IPU 2023.3 implemented.
·       IPU 2024.1 implemented.
·       IPU 2024.3 implemented.
